pycqed/measurement/pulse_sequences/multi_qubit_tek_seq_elts.py

```diff
@@ -74,8 +74,20 @@ def n_qubit_off_on(pulse_pars_list, RO_pars_list, return_seq=False,
         seg = segment.Segment('segment_{}'.format(i), pulses)
         seg_list.append(seg)
         seq.add(seg)
+
+    uhfs_used = dict()
+    for RO_pars in RO_pars_list:
+        uhf = RO_pars['I_channel'][:4]
+        if uhf not in uhfs_used.keys():
+            uhfs_used[uhf] = []
+        uhfs_used[uhf].append(1)
+
+    repeat_dict = {}
+    for uhf in uhfs_used:
+        repeat_dict[uhf] = ((1.0 + int(preselection))*len(pulse_combinations),1)
+
     if upload:
-        ps.Pulsar.get_instance().program_awgs(seq)
+        ps.Pulsar.get_instance().program_awgs(seq, repeat_dict=repeat_dict)
     if return_seq:
         return seq, seg_list
```

Comment:
needs to be updated to new way of using repeat dict :)

Reply:
Function was removed.